openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -113,6 +113,26 @@ export default {
             return records;
         },
 
+        async enhancedRecords(){
+            if (!this.records) return null;
+
+            const author_names = await get_author_names(this.records)
+
+            const enhanced_records = _.cloneDeep(this.records)
+
+            enhanced_records.flatMap(record =>
+                record.authors.map(entry=> {
```

Comment:
Since we're not returning anything, we can use forEach instead. Actually a for loop here might be the cleanest approach!

We can also avoid the extra loop by change the `get_author_names` method to return the full key with the `/authors/` prefix.

```suggestion
            for (const record of enhanced_records) {
                for (entry of record.authors) {
                    entry.author.name = author_names[entry.author.key];
                }
            }
```
```